packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -864,7 +875,14 @@ export class FluidDataStoreRuntime
 			}
 		}
 
-		return summaryBuilder.getSummaryTree();
+		const attachSummary = summaryBuilder.getSummaryTree();
+
+		// We need to include the GC Data so remote clients can learn of this DataStore's outbound routes
+		this.updateGCNodes(gcDataBuilder);
+		const gcData = gcDataBuilder.getGCData();
+		addBlobToSummary(attachSummary, gcDataBlobKey, JSON.stringify(gcData));
```

Comment:
@agarwal-navin  interested in your feedback on this bit of code (everything in this function)